### ISSUE TITLE
Extend README with external logging examples

### ISSUE THREAD

**Issue by r0b1ndot** [ansible/tower-operator#1086]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that AWX Operator is open source software provided for free and that I might not receive a timely response.

### Feature Summary

It would be great if the docs/Readme will be adjusted also for a fully working example for configuring external logging.
In our case, we also want to use external logging/ Logstash and also use a secret instead of putting the credentials plain to the extra settings.
Is this possible?

**Comment by tvo318** [comment]
Have you looked at this section of the admin guide? https://docs.ansible.com/automation-controller/latest/html/administration/logging.html

**Comment by r0b1ndot** [comment]
> Have you looked at this section of the admin guide? https://docs.ansible.com/automation-controller/latest/html/administration/logging.html

Yes thanks, i tried to configure with the Parameters doxumented but configuring via the Operator is different and variable handling is also Sometimes different, that's why I ASK since I expect that also other people would like to use this feature.
So adjust the Docs would be very useful.
